pypeit/slittrace.py

```diff
@@ -252,7 +255,17 @@ def _bundle(self):
         See :func:`pypeit.datamodel.DataContainer._bundle`. Data is
         always written to a 'SLITS' extension.
         """
-        return super(SlitTraceSet, self)._bundle(ext='SLITS', transpose_arrays=True)
+        bndl = super(SlitTraceSet, self)._bundle(ext='SLITS', transpose_arrays=True)
+        if self.maskdef_designtab is not None:
+            # save the table
+            tab_detached = bndl[0]['SLITS']['maskdef_designtab']
```

Comment:
@kbwestfall can comment on whether there is a more elegant way to do this.
Might not be.